python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -49,6 +49,44 @@
 PREEMPTIVE_RENEWAL_THRESHOLD = 0.9
 SHOTGUN_SSO_RENEWAL_INTERVAL = 5000
 
+# Some IdP will use JavaScript code which makes use of ES6. Our Qt4
+# environment is unfortunately missing some definitions which we
+# need to inject prior to running the IdP code.
+# The reference for this code is:
+#     https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind#Polyfill
+FUNCTION_PROTOTYPE_BIND_POLYFILL = """
+// Yes, it does work with `new funcA.bind(thisArg, args)`
```

Comment:
what's the significance of this comment? Can it be removed or clarified?

Reply:
The comment comes from the Mozilla polyfill code snipped, which I copied verbatim.
I did not want to modify that in any way.